modules/tinymce/changelog.txt

```diff
@@ -21,6 +21,8 @@ Version 5.5.0 (TBD)
     Fixed the `DOMUtils.split()` API incorrectly removing some content #TINY-6294
     Fixed pressing escape not focusing the editor when using multiple toolbars #TINY-6230
     Fixed the `dirty` flag not being correctly set during an `AddUndo` event #TINY-4707
+    Fixed `editor.selection.setCursorLocation` moving the cursor immediately outside pre elements in some circumstances #TINY-4058
+    Fixed the `br_in_pre` setting (when false) throwing an exception when pressing enter inside pre elements #TINY-4058
```

Comment:
I wonder if we should move the "when false" outside of brackets somehow... It is an awkward sentence to structure without the brackets though

Reply:
```
Fixed an exception being thrown when pressing enter inside pre elements while `br_in_pre` setting is false
```
How does that sound?